bin/make-zip.sh

```diff
@@ -21,52 +21,15 @@ output 2 "Creating archive... 🎁"
 
 ZIP_FILE=$1
 
-# Folders listed first then individual files
-zip -r ${ZIP_FILE} . \
-	-x \
-	.git/\* \
-	tests/\* \
-	bin/\* \
-	config/\* \
-	node_modules/\* \
-	vendor/bin/\* \
-	vendor/dealerdirect/\* \
-	vendor/doctrine/\* \
-	vendor/phar-io/\* \
-	vendor/phpcompatibility/\* \
-	vendor/phpdocumentor/\* \
-	vendor/phpspec/\* \
-	vendor/phpunit/\* \
-	vendor/sebastian/\* \
-	vendor/squizlabs/\* \
-	vendor/theseer/\* \
-	vendor/webmozart/\* \
-	vendor/woocommerce/\* \
-	vendor/wp-coding-standards/\* \
-	.distignore \
-	.editorconfig \
-	.gitignore \
-	.gitlab-ci.yml \
-	.travis.yml \
-	.DS_Store \
-	.zipignore \
-	Thumbs.db \
-	behat.yml \
-	circle.yml \
-	composer.json \
-	composer.lock \
-	Gruntfile.js \
-	package.json \
-	package-lock.json \
-	phpunit.xml \
-	phpunit.xml.dist \
-	multisite.xml \
-	multisite.xml.dist \
-	phpcs.xml \
-	phpcs.xml.dist \
-	README.md \
-	wp-cli.local.yml \
-	yarn.lock \
-	*.sql \
-	*.tar.gz \
-	*.zip
+build_files=$(ls dist/*/*.{js,css})
+
+zip -r ${ZIP_FILE} \
+	woocommerce-admin.php \
+	uninstall.php \
+	includes/ \
+	images/ \
+	$build_files \
+	languages/woocommerce-admin.pot \
+	readme.txt \
+	src/ \
+	vendor/
```

Comment:
I think there are some vendor files that we don't need to include in the release.  See https://github.com/woocommerce/woocommerce-admin/pull/3823/commits/760bcd346fd9c3195cdc9177b2b7320810a33fdf

Reply:
The vendor subfolders that were being excluded are not needed for the built plugin but do add to the size to the zip